content/content-tool/tool/src/java/org/sakaiproject/content/tool/SiteEmailNotificationDragAndDrop.java

```diff
@@ -624,43 +694,58 @@ private void addMessageTextDropbox(StringBuilder buf, String siteTitle, Referenc
 		}
 		if (contentHostingService.EVENT_RESOURCE_AVAILABLE.equals(function))
 		{
-			buf.append(rb.getFormattedMessage("db.text.new.dnd", new String[]{dropboxTitle, siteTitle, portalName, portalUrl}));
+			if (toplevel)
+			{
+				buf.append(rb.getFormattedMessage("db.text.new.dnd.toplevel", new Object[] {siteTitle, portalName, portalUrl}));
+			}
+			else
+			{
+				buf.append(rb.getFormattedMessage("db.text.new.dnd", new Object[] {dropboxTitle, siteTitle, portalName, portalUrl}));
+			}
 		}
 		else
 		{
-			buf.append(rb.getFormattedMessage("db.text.upd.dnd", new String[]{dropboxTitle, siteTitle, portalName, portalUrl}));
+			if (toplevel)
+			{
+				buf.append(rb.getFormattedMessage("db.text.upd.dnd.toplevel", new Object[] {siteTitle, portalName, portalUrl}));
+			}
+			else
+			{
+				buf.append(rb.getFormattedMessage("db.text.upd.dnd", new Object[] {dropboxTitle, siteTitle, portalName, portalUrl}));
+			}
 		}
 		buf.append(blankLine);
 
-		//FOR
-		for (int i=0;i<this.fileList.size();i++)
```

Comment:
Below changes are merely indentation modifications due to adding `null` check before the loop executes.